src/remix/templates.ts

```diff
@@ -5,7 +5,7 @@ export const ERROR_BOUNDARY_TEMPLATE_V2 = `const ErrorBoundary = () => {
 };
 `;
 
-export const HANDLE_ERROR_TEMPLATE_V2 = `function handleError(error, { request }) {
-  Sentry.captureRemixServerException(error, 'remix.server', request, true);
-}
+export const HANDLE_ERROR_TEMPLATE_V2 = `export const handleError = Sentry.wrapHandleErrorWithSentry((error, { request }) => {
```

Comment:
This generated the code below in my test app:

```
export export const handleError = Sentry.wrapHandleErrorWithSentry((error, { request }) => {
  // Custom handleError implementation
});
```

We should remove the logic that:

- `exports` the code in the template - [handle-error.ts
](https://github.com/getsentry/sentry-wizard/pull/621/files#diff-f2d0582b002c65e3ca8edfda1037ef827a172db2ba6f879b5d1d9abb42bfbf87R48)
- Or we can remove `export` from the template instead.